modulefiles/modulefile.ProdGSI.wcoss_c

```diff
@@ -38,18 +38,24 @@ module load prod_util/1.0.29
 module load PrgEnv-intel/5.2.56
 module unload intel/15.0.3.187
 module load intel/18.1.163
+#module load PrgEnv-intel/5.2.82
```

Comment:
Remove the commented out lines / clean up this modulefile too. Have it match what Russ put in the v16 release branch.

Reply:
Cleaned up this module file.